pnr/odin_II/meta.yaml

```diff
@@ -34,9 +34,11 @@ requirements:
     - xorg-libxcb
     - xorg-libxext
     - xorg-libxft
-    - tbb
-    - tbb-devel
+    - tbb <2021.0.0a0
```

Comment:
This seems like it will eventually bite us if another package requires the new version of tbb?  Why can the new package not but used?

Reply:
I think we want to be something like https://docs.conda.io/projects/conda-build/en/latest/resources/variants.html#pinning-at-the-recipe-level (e.g. `pin_compatible`)